src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/FieldSymbolReferenceFinder.cs

```diff
@@ -38,7 +38,7 @@ protected override bool CanFind(IFieldSymbol symbol)
             FindReferencesSearchOptions options,
             CancellationToken cancellationToken)
         {
-            return FindDocumentsAsync(project, documents, cancellationToken, symbol.Name);
+            return FindDocumentsAsync(project, documents, SupportsGlobalSuppression(symbol), cancellationToken, symbol.Name);
```

Comment:
you know it's a field... and you always return true for fields... so why do we need to call this helper?

Reply:
We can certainly do that. I thought `SupportsGlobalSuppression` could be a central documented place for symbol kinds that support global suppressions. Do you feel it is better to just inline this value at each call site?

Reply:
yes, please just inline.  the point of the individual finders is that we know exactly what we have.  
the benefit of a centralized place for ISymbol would be if we had a centralized location where we were just processing ISymbols and we needed to determine what to do there.